code/go/0chain.net/miner/protocol_block.go

```diff
@@ -624,12 +624,13 @@ func txnProcessorHandlerFunc(mc *Chain, b *block.Block) txnProcessorHandler {
 				zap.String("txn", txn.Hash), zap.Int32("idx", tii.idx),
 				zap.String("txn_object", datastore.ToJSON(txn).String()))
 		}
-		if ok, err := mc.ChainHasTransaction(ctx, b.PrevBlock, txn); ok || err != nil {
-			if err != nil {
-				tii.reInclusionErr = err
-			}
-			return false
-		}
+		//we can remove it, since it won't work, every accepted transaction will be included in chain and increment the nonce
+		//if ok, err := mc.ChainHasTransaction(ctx, b.PrevBlock, txn); ok || err != nil {
```

Comment:
I think this is for checking if the transaction will be executed twice, we should stop or report the txn if it was included in previous blocks.